Calling/ClientApp/src/components/Configuration.tsx

```diff
@@ -78,9 +78,12 @@ export default (props: ConfigurationScreenProps): JSX.Element => {
 
   const { groupId, setupCallClient, setGroup, unsupportedStateHandler } = props;
 
+  const memoizedSetupCallClient = useCallback(() => setupCallClient(unsupportedStateHandler), [
+    unsupportedStateHandler
+  ]);
   useEffect(() => {
-    setupCallClient(unsupportedStateHandler);
-  }, [setupCallClient, unsupportedStateHandler]);
+    memoizedSetupCallClient();
+  }, [memoizedSetupCallClient]);
```

Comment:
A bit confused here. `memoizedSetupCallClient` is only used in this `useEffect` block, and it only depends on `memoizedSetupCallClient`. So the original code here that didn't use a memoized callback should be identical to what you have (once you've added `setupCallClient` to the dependency array of `useCallback` above)?

Reply:
the props were the same value but were being created so it was causing the useEffect to get called multiple times. I had to memoize the callback so that it would only be created once and the useEffect would only try to set up the callClient once.